AnnoyancesFilter/sections/mobile-app.txt

```diff
@@ -15,6 +15,9 @@
 ##head[prefix~="medium-com:"] ~ body .js-openInAppButton
 ##head[prefix~="medium-com:"] ~ body .js-postActionsBarContent > .js-readNextInteractions
 !
+delishkitchen.tv##.footer-app-banner-wrap
```

Comment:
> I feel this rather makes layout ugly.

@Yuki2718 that's because of this `delishkitchen.tv##.footer-app-banner-wrap` rule. This should be fine.
```
delishkitchen.tv#$#aside[class^="footer-app-banner-wrap"] { margin: 1em 0 0 !important; visibility: hidden !important; height: 0 !important; }
```

Reply:
I think this rule is super good.
I changed to this.